tests/python/opinfo_input_generators.py

```diff
@@ -311,7 +311,11 @@ def cat_error_generator(op, dtype=torch.float32, requires_grad: bool = False, **
         "Unexpected number of dimensions",
     )
     # All tensors must have same shape except for the cat dimension
-    shape_mismatch = (([(2, 3), (4, 5)], 0), RuntimeError, "Tried to bind to a value")
+    shape_mismatch = (
+        ([(2, 3), (4, 5)], 0),
+        RuntimeError,
+        "a conflict was found with 2 different sizes",
```

Comment:
FYI, this PR accidentally fixes a bug. The current code constructs FusionExecutorCache with an empty fusion and populates it later. So FusionExecutorCache::exact_map_ is always empty. With this PR, the code will constructs a FusionExecutorCache with a populated fusion and therefore a proper LogicalDomainMap.